src/GraphQL.Tests/Types/EnumGraphTypeTests.cs

```diff
@@ -72,9 +72,9 @@ public void description_and_obsolete_from_enum()
         }
 
         [Fact]
-        public void adds_values_from_enum_custom_casing()
+        public void adds_values_from_enum_custom_casing_should_throw()
         {
-            type.ParseValue("rED").ShouldBe(Colors.Red);
+            Should.Throw<InvalidOperationException>(() => type.ParseValue("rED")).Message.ShouldBe("Unable to convert 'rED' to the scalar type 'Colors'");
```

Comment:
@Shane32 I believe that enums should parse exact values since GraphQL is case-sensitive.

Reply:
I agree

Reply:
This change should be noted in the migration document.  It may be important.

Reply:
I'm wondering if we shouldn't have a documented workaround here.  If someone had a very large GraphQL client written that does not consistently use the correct case, it could be absolutely necessary to have a workaround.

Reply:
I'll add migration notes but I would not invent workaround for such case. If someone had a very large GraphQL client he can post an issue.

Reply:
Ok. I would be prepared for such an issue.

Reply:
done